ontology/yaml/resources/units/units.yaml

```diff
@@ -102,6 +102,8 @@ distance:
   inches:
     multiplier: 0.0254
     offset: 0
+thermalefficiency:
+  kilowatts_per_ton: STANDARD
```

Comment:
The telemetry field looks to be on a scale of 0 to 100 and given that it is an efficiency I would suspect that the units should be percent - please confirm

Reply:
It should be kW/ton. I adjusted value range to all flexible between 0 and 10 kw/ton.